manifests/server.pp

```diff
@@ -7,6 +7,7 @@
 #   class rsync
 #
 class rsync::server(
+  $enable     = true,
```

Comment:
Could you rename this to `service_ensure` to more closely follow other modules that do this? Also the name `ensure` would more closely match puppet's parlance than `enable` does.

It should also accept the values `'running'` and `'stopped'` and convert that to `true/false/'present'/'absent'` as needed.